### ISSUE TITLE
Character voice and appearance selection not saving after first initialization

### ISSUE THREAD

**Issue by kurazarrh** [anegostudios/VintageStory-Issues#2249]
**Game Version:** 1.17.8
**Platform:** Windows   
**Modded:** No  
**SP/MP:** Singleplayer  

### Description
When using .charsel to change the player character's voice, it does not persist between restarts (and sometimes changes back to the original selection during the current session).

### How to reproduce
1. Start a new single-player game.
2. During the character selection, set the voice to "Medium" and "Trumpet".
3. In a single-player game, hit enter, type ".charsel" without quotes, and hit enter again.
4. Select the character's voice option: change both dropdowns to something other than "Medium" and "Trumpet".
5. Confirm the character's skin and class (this will require selecting the correct class again, as the original selection does not persist).
6. Perform an action that causes the character to make a sound, like typing ".wave" into chat and submitting (to confirm that the voice has changed).
7. Save, close the world, and exit the game.
8. Start the game client again and reload the same world.
9. Type ".wave" into the chat again (or do another action that causes the character to use their voice.
10. The voice is back to Medium Trumpet.

### Expected behavior
The character's voice should always reflect the user's latest selection in the .charsel screen.

**Comment by kurazarrh** [comment]
Updating this bug as of 1.18.1 to include that when .charsel is used AFTER initial character creation, the only change that is saved to the character is their class. Changing skin/eyes/hair/voice in .charsel does nothing, and your original character attributes are not overwritten.

**Comment by kurazarrh** [comment]
@tyronx Not sure what the etiquette is for pinging you about bugs, but it looks like no one has looked at this bug yet. Wanted to bring it to your attention, since it makes the .charsel command only work for changing the player's class. Can confirm this is still an issue in 1.18.6 - I chose the wrong color for my character's skin, and even though it shows up correctly in the screen when I do a .charsel, once I complete the process, my character's skin (and eyes/hair/eyes/voice, etc) are all stuck at what I chose originally.
